### ISSUE TITLE
Part of Speech Tagging

### ISSUE THREAD

**Issue by browndw** [guillaume-be/rust-bert#139]
First, thanks so much for this. I'm really enjoying using these resources. I just had a question regarding part of speech tagging. As I understand it, both named entity recognition and part of speech tagging are token classification tasks. As such, their processing pipelines will be very similar. I've had success running NER with rust-bert. However, I can't quite get POS tagging to work. In the documentation, The NER example is given as:

let ner_model = NERModel::new(default::default())?;
let input = ["My name is Amy. I live in Paris.", "Paris is a city in France."];
let output = ner_model.predict(&input); 

And the POS example as:

let ner_model = NERModel::new(default::default())?;
let input = ["My name is Bob];
let output = ner_model.predict(&input);

Are these identical? And if so, how/where does one specify that the label output should be a POS tag?

**Comment by guillaume-be** [comment]
Hello @browndw ,

You are right that both pipelines are very similar. The only differences are a different default model architecture and post-processing pipeline. This is a copy/paste mistake on my end for the documentation, I am pushing an update to fix this (https://github.com/guillaume-be/rust-bert/pull/140).

The correct way to create an call a POS model should be:
```rust
use rust_bert::pipelines::pos_tagging::POSModel;

fn main() -> anyhow::Result<()> {
  let pos_model = POSModel::new(Default::default())?;
  let input = [
    "My name is Bob",
  ];
  let output = pos_model.predict(&input);
}
```
returning:
```
[
    POSTag {
        word: String::from("My"),
        score: 0.1560,
        label: String::from("PRP"),
    },
    POSTag {
        word: String::from("name"),
        score: 0.6565,
        label: String::from("NN"),
    },
    POSTag {
        word: String::from("is"),
        score: 0.3697,
        label: String::from("VBZ"),
    },
    POSTag {
        word: String::from("Bob"),
        score: 0.7460,
        label: String::from("NNP"),
    },
]
```

**Comment by browndw** [comment]
Thanks so much! Got it working. But just FYI, I had to set the dependency to the **github** repository rather than the default **crates.io**. That crate doesn't include **pos_tagging.rs** in the pipelines directory (or at least its path doesn't appear in the **deps** folder preventing it from compiling). And again, thanks for the terrific resource.

**Comment by guillaume-be** [comment]
Glad it worked out!

Yes - the POS tagging has not been released yet (changelogs are available at https://github.com/guillaume-be/rust-bert/blob/master/CHANGELOG.md). I am planning a new release this weekend so you should be able to pull it from crates.io by then.

**Comment by guillaume-be** [comment]
POS Tagging now published in v0.15.0
